File: sample/app-java/src/main/java/com/appice/sample/receiver/CampaignCampReceiver.java
package com.appice.sample.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import semusi.activitysdk.ContextSdk;

public class CampaignCampReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Bundle extras = intent.getExtras();
        if (extras != null) {
            System.out.println("appICE : from appice " + extras.getString("ai_content"));
            ContextSdk.pushNotificationClicked(extras, context);
        }
    }
}

File: sample/app-kotlin/src/main/java/com/semusi/receiver/CampaignCampReceiver.kt
package com.semusi.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

import semusi.activitysdk.ContextSdk


class CampaignCampReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context?, intent: Intent) {
        val extras = intent.extras
        if (extras != null) {
            System.out.println("appICE : from appice " + extras.getString("ai_content"))
            ContextSdk.pushNotificationClicked(extras, context)
        }
    }
}